### PR TITLE
Initial attempt to lerp white color on symbols

### PR DESCRIPTION
**Pull request by JamesComo44**
Looks pretty sexy

### CODE REVIEW

File: DigitalRain/Raindrops/StandardRaindrop.cs [hunk 1]
```diff
@@ -58,30 +62,33 @@ public char GetSymbolFromPool()
             return SymbolPool[index];
         }
 
-        // IRaindrop
+        float lerpScaleFactor = 350;
+        float lerpAmount = 400;
```
Comment (JamesComo44) [inline DigitalRain/Raindrops/StandardRaindrop.cs:64]: OK BEFORE YOU FREAK OUT!

I am working on getting this rolled into the config class. But I am currently modifying that class in my `global_config_access` branch. So I wanted to merge this first so the changes are more visible.

**Comment by JamesComo44** [comment]
Proof of sexy look
image

File: DigitalRain/Program.cs [hunk 2]
```diff
@@ -14,8 +14,8 @@ static void Main(string[] args)
                 var configFileName = args[configFileIndex];
                 var configReader = new ConfigReader();
                 var config = configReader.ReadConfig(configFileName);
-                using (var game = new DigitalRainGame(config))
-                    game.Run();
+                using var game = new DigitalRainGame(config);
+                game.Run();
```
Comment (ghost) [inline DigitalRain/Program.cs:18]: Why this change?  I don't know what the new version is doing exactly but it looks very different from the old version, which I think is cleaning up the game object

File: DigitalRain/Raindrops/StandardRaindrop.cs [hunk 3]
```diff
@@ -36,6 +37,7 @@ public StandardRaindrop(ColumnSpace space, double lifeSpan, Color symbolColor)
             LifeRemaining = lifeSpan;
 
             _symbolColor = symbolColor;
+            _startColor = Color.White;
```
Comment (ghost) [inline DigitalRain/Raindrops/StandardRaindrop.cs:40]: Eventually it'll be cool to play with different colors via the config

File: DigitalRain/Raindrops/StandardRaindrop.cs (hunk 1 above)
Comment (ghost) [reply DigitalRain/Raindrops/StandardRaindrop.cs:64]: Hmmm again I don't think we need both these values.
It looks like the lerp method takes a percentage (between 0 and 1).  So I imagine you just want to map that to a timeframe much like we did with lifespan to alpha.  So if I want this effect to last for 3 seconds, at t=0 I want a lerpAmount of 0.0 and at t >= 3 I want a lerpAmount of 1.0.

File: DigitalRain/Program.cs (hunk 2 above)
Comment (JamesComo44) [reply DigitalRain/Program.cs:18]: Recommended by C# linter

File: DigitalRain/Raindrops/StandardRaindrop.cs (hunk 3 above)
Comment (JamesComo44) [reply DigitalRain/Raindrops/StandardRaindrop.cs:40]: That was the goal! But then I got distracted working on `global_config_access` branch

File: DigitalRain/Raindrops/StandardRaindrop.cs (hunk 1 above)
Comment (JamesComo44) [reply DigitalRain/Raindrops/StandardRaindrop.cs:64]: Feel free to change this! This was the best I could do with my smooth brain
